scss/_base_code.scss

```diff
@@ -0,0 +1,28 @@
+// Base code styles
+@mixin vf-b-code {
+  code,
+  samp,
+  kbd {
+    font-family: $font-monospace;
+    text-align: left;
+  }
+
+  pre,
+  code {
+    text-align: left;
+    white-space: pre-line;
```

Comment:
Gotcha - this was already there but I'll remove it now and then merge as other points have been addressed as Proposals for WG.